PasswordVerifier.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your password: ");
+            string password;
+            password = Console.ReadLine();
+            bool isTrue = true;
```

Comment:
rename this variable to `isValid`

Reply:
Done! Resolved on the new commit. Thank you!